src/components/CardAdministration.vue

```diff
@@ -41,19 +41,22 @@
             @click="toggleParams($event, assessmentId)"
           />
         </span>
-        <div v-if="showParams">
-          <PvOverlayPanel v-for="assessmentId in assessmentIds" :key="assessmentId" :ref="paramPanelRefs[assessmentId]">
-            <PvDataTable
-              striped-rows
-              class="p-datatable-small"
-              table-style="min-width: 30rem"
-              :value="toEntryObjects(params[assessmentId])"
-            >
-              <PvColumn field="key" header="Parameter" style="width: 50%"></PvColumn>
-              <PvColumn field="value" header="Value" style="width: 50%"></PvColumn>
-            </PvDataTable>
-          </PvOverlayPanel>
-        </div>
+        <PvOverlayPanel v-for="assessmentId in assessmentIds" :key="assessmentId" :ref="paramPanelRefs[assessmentId]">
+          <div v-if="props.assessments && (assessment = getAssessment(assessmentId))">
```

Comment:
Is the `props.assessments` necessary in the `v-if` here. The `PvOverlayPanel` already has a `v-for` over the `assessmentIds`, which is a computed value that depends on `props.assessments` being defined. Also, my understanding is that one does not have to explicitly reference `props.` in the template portion.

Reply:
Actually, I don't really understand this `v-if`. Why is it needed? If the point is to set the value of a variable called `assessment` to `getAssessment(assessmentId)`, why not just use `getAssessment(assessmentId)` below.

Alternatively, we could adjust the `v-for` loop above to loop over `assessments` instead of `assessmentIds`.

Reply:
you are right, It was to check first if we have the id and name, and assigned to a variable. But yes, I can take it out